api/src/main/java/pl/kolendateam/dadcard/characterCard/entity/Character.java

```diff
@@ -136,11 +146,39 @@ public void addSavingThrowLevelOne(ClassPc classPc){
             this.savingThrow.setWill(this.savingThrow.getWill()+bonusWill);            
     }
 
+    public void minusSavingThrowLevelOne(ClassPc classPc){
+
+        String stringSavingThrow = classPc.getSavingThrow();
+    
+            double bonusFortitude;
+            if(stringSavingThrow.charAt(0) == ValueEnum.HIGH.getValueEnum().charAt(0)){
```

Comment:
w takich przypadkach fajnie sie sprawdza conditional operator

Reply:
racją że jeżeli są tylko 2, to użyje conditionala

Reply:
może tak jest bardzej czysty?